src/@context/Asset.tsx

```diff
@@ -25,6 +25,7 @@ export interface AssetProviderValue {
   title: string
   owner: string
   error?: string
+  warning?: string
```

Comment:
It’s a nice touch to differentiate between warning & error in UI, but in this case the asset the user looked for and not being shown is always an error. So fine if we handle all this in existing `error` state

Reply:
Some small leftover here, and further down `warning` state is still defined

Reply:
> Some small leftover here, and further down `warning` state is still defined

I've removed the unused states, but I can't remove line 28 because if I remove `warning` from the AssetProviderValues then the [`AssetDetails`](https://github.com/oceanprotocol/market/blob/fix/issue-1600-proper-state-check-asset/src/components/Asset/index.tsx#L12) breaks. If we remove it from the provider, we'll need to remove it completely.

Reply:
Ah yes, also remove from `components/Asset/index.tsx`, then nothing should break as we are not using this newly introduced `warning` state